batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/service/EmployeeService.java

```diff
@@ -0,0 +1,44 @@
+package com.lftechnology.batch7crud.service;
+
+import com.lftechnology.batch7crud.dao.EmployeeDao;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+
+import javax.xml.crypto.Data;
+import java.sql.SQLException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.sql.SQLException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)